package.json

```diff
@@ -5,10 +5,11 @@
   "dependencies": {
     "@1hive/1hive-ui": "^1.0.1",
     "@1hive/connect-thegraph-conviction-voting": "^0.1.7",
+    "@1hive/uniswap-sdk": "^1.0.0",
     "@aragon/connect": "^0.3.0",
     "bignumber.js": "^9.0.0",
     "clipboard-polyfill": "^2.8.6",
-    "ethers": "^5.0.2",
+    "ethers": "4.0.28",
```

Comment:
Bote that we needed to downgrade ethers version so it could work with uniswap v1 sdk